jetty-security/src/main/java/org/eclipse/jetty/security/ConstraintSecurityHandler.java

```diff
@@ -409,7 +409,8 @@ public void addRole(String role)
     @Override
     protected void doStart() throws Exception
     {
-        _constraintMappings.stream().forEach(m -> processConstraintMapping(m));
+        _constraintMap.reset();
```

Comment:
Why is this back in? It is reset in doStop()

Reply:
Just being safe